doc/user_guide/ecosystem.rst

```diff
@@ -1,20 +1,131 @@
+
+.. List of links.
+.. _`Data Visualization Course`: https://viz-learn.mds.ubc.ca
+.. _`Key Capabilities for Data Science`: https://extendedlearning.ubc.ca/programs/key-capabilities-data-science
+
+`Brief Introduction Videos`_
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Calmcode provides a few videos that give a brief overview of the Altair library.
+
+.. List of links.
+.. _`Brief Introduction Videos`: https://calmcode.io/altair/introduction.html
+
+
+.. _altair-ecosystem:
+
+Altair Ecosystem
+----------------
+
+This is a list of projects directs related to Altair. There are many other packages that can be used in tandem with Altair, e.g. `dashboard packages which you can read more about in the answers to this StackOverflow question`_.
+
+.. List of links.
+.. _`dashboard packages which you can read more about in the answers to this StackOverflow question`: https://stackoverflow.com/questions/49833866/making-dashboards-using-altair
+
+Vega-Lite_
+~~~~~~~~~~
+
+The higher-level visualization grammar that Altair implements in Python.
+
+.. List of links.
+.. _Vega-Lite: https://vega.github.io/vega-lite
+
+vl-convert_
+~~~~~~~~~~~
+
+
+VegaFusion_
+~~~~~~~~~~~
+
+VegaFusion provides server-side scaling for Altair charts, which can accelerate interactive charts, extract transformed data, and perform data-intensive aggregations on the server and prune unused columns from the source datase yielding smaller size visualizations
+
+.. List of links.
+.. _VegaFusion: https://vegafusion.io/
+
+altair_saver_
+~~~~~~~~~~~~~
+
+Enables saving charts to a variety of output types.
+
+.. List of links.
+.. _altair_saver: https://github.com/altair-viz/altair_saver
+
+altair_data_server_
+~~~~~~~~~~~~~~~~~~~
+
+Data transformer plugin that transparently serves data for charts.
+
+.. List of links.
+.. _altair_data_server: https://github.com/altair-viz/altair_data_server
+
+altair_pandas_
+~~~~~~~~~~~~~~
+
+Altair backend for the pandas plotting API.
+
+.. List of links.
+.. _altair_pandas: https://github.com/altair-viz/altair_pandas
+
+vega_datasets_
+~~~~~~~~~~~~~~
+
+Offline access to the Vega datasets used in the Altair documentation.
+
+.. List of links.
+.. _vega_datasets: https://github.com/altair-viz/vega_datasets
 
 altair_recipes_
----------------
+~~~~~~~~~~~~~~~
 
-altair-recipes provides a collection of ready-made statistical graphics for vega. See the `docs <https://altair-recipes.readthedocs.io/en/latest/>`__.
+altair_recipes provides a collection of ready-made statistical graphics for Altair. See the `docs <https://altair-recipes.readthedocs.io/en/latest/>`__.
 
 .. List of links.
 .. _altair_recipes: https://github.com/piccolbo/altair_recipes
 
 gpdvega_
```

Comment:
Is this package still of use? I'm not familiar with it so maybe I miss something. Last commit on GitHub is from 2018 and it's main feature seems to be support for GeoPandas dataframes which was added to Altair in 3.3.0 (2019). I was able to execute the examples from the documentation of gpdvega without having it installed or importing it.

Can't make a code suggestion here as the diff range is too short.

Reply:
Good point, I think we can remove it since the Altair docs now nicely shows how to work with geodata

Reply:
Removed